src/inputs/CheckboxGroup.tsx

```diff
@@ -33,29 +34,44 @@ export interface CheckboxGroupProps {
   onFocus?: () => void;
   /** Number of columns to display checkboxes */
   columns?: number;
-  /** Hide label if it is not needed */
-  hideLabel?: boolean
 }
 
 export function CheckboxGroup(props: CheckboxGroupProps) {
-  const { options, label, values, errorMsg, helperText, onBlur, onFocus, columns = 1, hideLabel = false } = props;
+  const { fieldProps } = usePresentationContext();
+  const {
+    options,
+    label,
+    labelStyle = fieldProps?.labelStyle ?? "above",
+    values,
+    errorMsg,
+    helperText,
+    onBlur,
+    onFocus,
+    columns = 1,
+  } = props;
 
   const state = useCheckboxGroupState({ ...props, value: values });
   const { groupProps, labelProps } = useCheckboxGroup(props, state);
   const tid = useTestIds(props);
 
   return (
     <div {...groupProps} onBlur={onBlur} onFocus={onFocus} {...tid}>
-     {!hideLabel && <Label label={label} {...labelProps} {...tid.label} />}
-      <div css={Css.dg.gtc(`repeat(${columns}, auto)`).gap2.$}>
-        {options.map((option) => (
-          <CheckboxGroupItem
-            key={option.value}
-            {...option}
-            groupState={state}
-            selected={state.value.includes(option.value)}
-          />
-        ))}
+      <div css={Css.if(labelStyle === "left").df.fdc.add("flexFlow", "wrap").$}>
+        {labelStyle !== "hidden" && (
+          <div css={Css.if(labelStyle === "left").mw50.my("auto").$}>
```

Comment:
@bdow I think this is causing the dom diff (in this case it does need the vertically centered label to match the figma design)